src/Module/BaseApi.php

```diff
@@ -101,6 +104,84 @@ public function run(ModuleHTTPException $httpException, array $request = [], boo
 		return parent::run($httpException, $request);
 	}
 
+	/**
+	 * Processes data from GET requests and sets paging conditions
+	 *
+	 * @param array $request       Custom REQUEST array
+	 * @param array $condition     Existing conditions to merge
+	 * @return array paging data condition parameters data
+	 * @throws \Exception
+	 */
+	public function addPagingConditions(array $request, array $condition): array
+	{
+		$requested_order = $request['friendica_order'];
+		if ($requested_order == TimelineOrderByTypes::ID) {
+			if (!empty($request['max_id'])) {
+				$condition = DBA::mergeConditions($condition, ["`uri-id` < ?", intval($request['max_id'])]);
+			}
+
+			if (!empty($request['since_id'])) {
+				$condition = DBA::mergeConditions($condition, ["`uri-id` > ?", intval($request['since_id'])]);
+			}
+
+			if (!empty($request['min_id'])) {
+				$condition = DBA::mergeConditions($condition, ["`uri-id` > ?", intval($request['min_id'])]);
+			}
+		} else {
+			switch ($requested_order) {
+				case TimelineOrderByTypes::CREATED:
+					$order_field = 'created';
+					break;
+				default:
+					$order_field = 'uri-id';
+			}
+			if (!empty($request['max_id'])) {
+				$condition = DBA::mergeConditions($condition, ["`$order_field` < ?", DateTimeFormat::convert($request['max_id'], DateTimeFormat::MYSQL)]);
+			}
+
+            if (!empty($request['since_id'])) {
+                $condition = DBA::mergeConditions($condition, ["`$order_field` > ?", DateTimeFormat::convert($request['since_id'], DateTimeFormat::MYSQL)]);
+            }
```

Comment:
Mixed indentation again.
```suggestion
			if (!empty($request['since_id'])) {
				$condition = DBA::mergeConditions($condition, ["`$order_field` > ?", DateTimeFormat::convert($request['since_id'], DateTimeFormat::MYSQL)]);
			}
```